src/include/simeng/models/emulation/Core.hh

```diff
@@ -34,12 +38,16 @@ class Core : public simeng::Core {
   const ArchitecturalRegisterFileSet& getArchitecturalRegisterFileSet()
       const override;
 
+  /** Send a syscall to the system syscall handler. */
+  void sendSyscall(const OS::SyscallInfo syscallInfo) const override;
```

Comment:
Same as [this](https://github.com/UoB-HPC/SimEng/pull/285/files#r1113405601) comment

Reply:
Dealt with similarly to referenced comment

Reply:
It still says system while the one in simeng/Core.hh says simulated Operating System's

Reply:
outdated